tests/test_dir_layouts.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+from pathlib import Path
+import shutil
+import bids
+from petdeface.petdeface import PetDeface
+from os import cpu_count
+from bids.layout import BIDSLayout
+
+import tempfile
+
+# collect test bids dataset from data directory
+data_dir = Path(__file__).parent.parent / "data"
```

Comment:
Should we consider downsampling the test data to speed up the tests?

Reply:
From my understanding it's a registration issue with the current test data, either way I think we should optimize later and push through with it now.